draft-js-plugins-editor/src/index.d.ts

```diff
@@ -105,6 +105,8 @@ export interface PluginEditorProps extends EditorProps {
   defaultKeyBindings?: boolean;
   defaultKeyCommands?: boolean;
   defaultBlockRenderMap?: boolean;
+  
+  keyBindingFn?(e: SyntheticKeyboardEvent): EditorCommand | null | undefined;
```

Comment:
It should be altered up on line 50 IMO, aside from that +1

 https://github.com/draft-js-plugins/draft-js-plugins/pull/1396/commits/14ebc8b4a777bfe53433acf2bf74cc9675283c20#diff-b2ce8263c95dfe11ec41b895dc569ac6R50
```suggestion
  keyBindingFn?(
    e: KeyboardEvent,	
    pluginFunctions: PluginFunctions
  ): DraftEditorCommand | null | void;
```

Reply:
Hi @adamdharrington, I think you are right it should be added to line 50, yet it also needs to remain here as PluginEditorProps, does not inherit from EditorPlugin